frontend/src/components/repartitions/Groups.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Container from "react-bootstrap/Container";
+import Card from "react-bootstrap/Card";
+import { NewRepartition } from "./NewRepartition";
+
+const titles = ["titre A", "titre B"];
+
+function titlecards() {
+  return (
+    <Container className="mt-5">
+      {titles.map((title) => (
+        <Card className="text-left">
+          <Card.Title>{title}</Card.Title>
+        </Card>
+      ))}
+    </Container>
+  );
+}
+
+export const Groups = () => {
+  return <NewRepartition>{titlecards()}</NewRepartition>;
+};
```

Comment:
1. If your function is pure, you don't need to use `return`, simply replace `{}` by `()` and remove `return`
2. Since `Titlecards` is now officially  a component (see comment above), simply call it with its tag
```suggestion
export const Groups = () => (
 <NewRepartition><Titlecards /></NewRepartition>
);
```